src/app/help/help.component.ts

```diff
@@ -24,4 +26,12 @@ export class HelpComponent {
         this.version = res.version;
       });
   }
+
+  showSection(whichSection) {
+    if (whichSection === this.currentSection){
```

Comment:
`this.currentSection = whichSection === this.currentSection ? whichSection : ''`?